polyval/Cargo.toml

```diff
@@ -22,6 +22,7 @@ zeroize = { version = "0.10", optional = true, default-features = false }
 hex-literal = "0.1"
 
 [features]
+ghash = []
```

Comment:
Alternatively we could use the `ghash` crate for this

Reply:
Hm, I think using `ghash` crate would be better for visibility, but then it would be probably better to move `field` module to a separate crate. If you prefer the current way I am fine with it as well.

Reply:
How about we leave them in the same crate for now, and I can investigate things getting VPCLMULQDQ working, and runtime CPU detection and fallback to the software implementation, and once all of that stuff is working we can investigate splitting them into separate crates?

Reply:
Yes, sounds good to me.